### PR TITLE
feat(translation): add Basque translation

### PR DESCRIPTION
**Pull request by jcarral**
Basque language added to translations settings

| Question         | Response    |
| ---------------- | ----------- |
| Version?         | v1.4.0      |
| Devices tested?  | iPhone 7... |
| Bug fix?         | no      |
| New feature?     | yes      |
| Includes tests?  | no      |
| All Tests pass?  | yes      |
| Related ticket?  | #676       |

## Description
Basque translation added

### CODE REVIEW

File: ios/GitPoint.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -5,7 +5,6 @@
 	};
 	objectVersion = 46;
 	objects = {
-
```
Comment (andrewda) [inline ios/GitPoint.xcodeproj/project.pbxproj]: Don't push changes to this

File: .all-contributorsrc [hunk 2]
```diff
@@ -655,6 +655,16 @@
...
+    },
+    {
+      "login": "jcarral",
+      "name": "Joseba Carral",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/13391685?v=4",
+      "profile": "http://codevs.es",
+      "contributions": [
+        "code",
+        "plugin"
+      ]
```
Comment (machour) [inline .all-contributorsrc]: Use `translation` instead of `code` and `plugin`

Comment (jcarral) [reply .all-contributorsrc]: Yes, sorry, i did it quickly. You're right it's translation 😅

File: ios/GitPoint.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jcarral) [reply ios/GitPoint.xcodeproj/project.pbxproj]: I didn't notice the change, my editor made it automatically. I hope now it's ok. Newbie mistake

File: src/locale/languages/eu.js [hunk 3]
```diff
@@ -0,0 +1,345 @@
+export const eu = {
+  auth: {
+    login: {
+      connectingToGitHub: 'GitHubera konektatzen...',
+      preparingGitPoint: 'GifPoint prestatzen...',
```
Comment (ocarreterom) [inline src/locale/languages/eu.js]: GitPoint

**Pull request merged**